index.js

```diff
@@ -68,7 +68,39 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("#header")
+
+header.addEventListener("click", function(){
+  toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
 
+const button = document.querySelector(".like-button")
+const likes = document.querySelector(".likes")
+
+button.addEventListener("click", function(){
+  const likesCount = parseInt(likes.textContent)
+  likes.textContent = likesCount + 1 + " Likes"
+})
+
 
 /***** Deliverable 3 *****/
+
+const form = document.querySelector("#new-goal-form")
+
+form.addEventListener("submit", function(event){
+  event.preventDefault()
+  const link = event.target.link.value
+  const description = event.target.description.value 
+
+  const newGoal = {
+    id: "",
+    link: link,
+    description: description
+  }
+
+  renderGoal(newGoal)
```

Comment:
Well done! Once we get to working with fetch, the server will take care of making the ID for us, so don't worry about that part too much. 

If you want to save on variables declarations, you could also create your object like so:

```js
  const newGoal = {
    id: "",
    link: event.target.link.value,
    description: event.target.description.value
  }
```